src/structs/vpx_bool_writer.rs

```diff
@@ -99,15 +100,12 @@ impl<W: Write> VPXBoolWriter<W> {
         if value {
             *tmp_value += split;
             *tmp_range -= split;
-
-            shift = (*tmp_range as u8).leading_zeros() as i32;
         } else {
             *tmp_range = split;
-
```

Comment:
I have a cleaner version of this in the cabac project
https://github.com/mcroomp/cabac/blob/77be08aadf92b8fc6d9cd37d17cec0fcef55605b/src/vp8.rs#L366

completely eliminates adjusting the carry in the buffer

Reply:
Nice, I'll apply this thing.

Reply:
Will be applied in the next PR on bit writer.

Reply:
Ok if you could remove the bool writer changes here and I can approve. Thanks!

Reply:
Removed - only changes about buffer flashing remain.

Reply:
The buffer flushing is unnecessary with the new bool_writer changes since it doesn't require a separate buffer anymore at all.

Reply:
Yes, but this will be in the next PR :)